src/components/Task/TaskInput/TaskInput.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+
+class TaskInput extends Component {
+    state = {
+        input: ''
+    }
```

Comment:
try to use best practises for class-components: use constructor(), bind method, keep render() as simple as possible